apis/fluent-en_GB/atrium-api-fluent-en_GB-jvm/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/pathExpectations.kt

```diff
@@ -115,6 +115,26 @@ fun <T : Path> Expect<T>.notToExist(): Expect<T> =
 fun <T : Path> Expect<T>.toBeReadable(): Expect<T> =
     _logicAppend { isReadable() }
 
+/**
+ * Expects that the subject of `this` expectation (a [Path]) is not readable;
+ * meaning that there is a file system entry at the location the [Path] points to and
+ * that the current thread does not have the permission to read from it.
+ *
+ * This assertion _resolves_ symbolic links.
+ * Therefore, if a symbolic link exists at the location the subject points to,
+ * search will continue at the location the link points at.
+ *
+ * This assertion is not atomic with respect to concurrent file system operations on the paths the assertion works on.
+ * Its result, in particular its extended explanations, may be wrong if such concurrent file system operations
+ * take place.
+ *
+ * @return an [Expect] for the subject of `this` expectation.
+ *
```

Comment:
Can you please also create a corresponding sample in ch.tutteli.atrium.api.fluent.en_GB.samples.PathExpectationSamples and add the `@sample` link here:
```suggestion
 * @sample ch.tutteli.atrium.api.fluent.en_GB.samples.PathExpectationSamples.notToBeReadable
 *
```